src/servers/src/mysql/writer.rs

```diff
@@ -149,7 +149,7 @@ impl<'a, W: AsyncWrite + Unpin> MysqlResultWriter<'a, W> {
         error: Error,
         w: QueryResultWriter<'a, W>,
     ) -> Result<()> {
-        error!(error; "Failed to execute query '{}'", query);
+        error!("Failed to execute query '{}', error:{:?}", query, error);
```

Comment:
Why change this?

Reply:
Nice catch! This is added for debugging. I forget to revert it.